x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_google_workspace_admin_role_assigned_to_user.json

```diff
@@ -50,5 +50,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 8
+  "version": 10
```

Comment:
Version already bumped from `6` to `8` in previous `8.2 Rules` PR, so looks to be a version jump from `6` to `10` in `8.2` without any content changes??

https://github.com/elastic/kibana/pull/129526/files#diff-76bfd3d11ee6aec9d3b671c17129dcf28b5b59b644d8942a75f42550a0111187R53

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files